README.md

```diff
@@ -27,19 +27,23 @@ Install the latest version of `confluent` to `/usr/local/bin`:
 
     brew install confluentinc/tap/cli
 
-#### Scripted installation
+#### APT (Ubuntu and Debian)
 
-Install the latest version of `confluent` to `/usr/local/bin`:
-
-    curl -sL https://raw.githubusercontent.com/confluentinc/cli/main/install.sh | sh -s -- -b /usr/local/bin
+Install the latest version of `confluent` to `/usr/bin` (requires `glibc 2.17` or above for `amd64` and `glibc 2.27` or above for `arm64`):
 
-Print a complete list of versions available for download:
+    wget -qO - https://packages.confluent.io/confluent-cli/deb/archive.key | sudo apt-key add -
+    sudo apt install software-properties-common
```

Comment:
do these installs need a confirm from prompt?

Reply:
Oh, right. The apt one does; I'll have to double check on the yum one.